src/BaseServer.ts

```diff
@@ -205,7 +224,7 @@ export class BaseServer {
 
         // handle collections
         const matches = request.url?.match(
-            new RegExp(`^${this.baseUrl}\\/([^\\/?]+)(\\/(\\d+))?(\\?.*)?$`),
+            new RegExp(`^${this.baseUrl}\\/([^\\/?]+)(\\/(\\w))?(\\?.*)?$`),
```

Comment:
I'm afraid this may break things

Reply:
I haven't noticed any and I don't see any other way to support custom ids